content/automate/ManageIQ/Service/Retirement/StateMachines/ServiceRetirement.class/__methods__/update_service_retirement_status.rb

```diff
@@ -6,6 +6,7 @@
 server = $evm.root['miq_server']
```

Comment:
@d-m-u This is the correct file name, so you'll want to copy your changes from update_serviceretirement_status.rb in this PR to this file and remove the  update_serviceretirement_status.rb file from this PR. ...ment/StateMachines/ServiceRetirement.class/__methods__/update_serviceretirement_status.rb